resources/views/layouts/parts/navbar.twig

```diff
@@ -92,14 +92,16 @@
                     </li>
                 {% endif %}
 
-                <li class="nav-item dropdown">
-                    <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-                        {{ m.icon('translate') }}
-                    </a>
-                    <ul class="dropdown-menu dropdown-menu-end">
-                        {{ menuLanguages()|join(" ")|raw }}
-                    </ul>
-                </li>
+                {% if showLanguagesMenu() %}
```

Comment:
As this function is only used once i would just directly use `config('locales') | length > 1`